packages/lib/delegationCredential/server.ts

```diff
@@ -443,10 +474,28 @@ export const enrichUserWithDelegationCredentialsWithoutOrgId = async <
   };
 };
 
+export const enrichUserWithDelegationCredentials = async <
```

Comment:
Added a fn that by default won't include delegatedTo which holds the serviceAccount key